applications/stepswitch/src/stepswitch_resources.erl

```diff
@@ -708,6 +718,17 @@ gateway_to_endpoint(DestinationNumber
         ,{<<"Outbound-Caller-ID-Number">>, CIDNumber}
         ,{<<"Outbound-Caller-ID-Name">>, CIDName}
         ,{<<"SIP-Invite-Parameters">>, sip_invite_parameters(Gateway, OffnetJObj)}
+        ,{<<"Privacy-Method">>, PrivacyMethod}
+        ,{<<"Privacy-Hide-Name">>
```

Comment:
Could these be nested keys under a `Privacy` top-level key instead? Then you could do `Should-Hide-Name` and `Should-Hide-Number` which is more self-evident that they are boolean toggles.